packages/scandipwa/src/component/ProductTabs/ProductTabs.component.js

```diff
@@ -21,76 +21,75 @@ import './ProductTabs.style';
 /** @namespace Component/ProductTabs/Component */
 export class ProductTabs extends PureComponent {
     static propTypes = {
-        tabNames: PropTypes.arrayOf(PropTypes.string).isRequired,
-        children: PropTypes.node,
-        defaultTab: PropTypes.string
+        tabs: PropTypes.arrayOf(PropTypes.shape({
+            name: PropTypes.string.isRequired,
+            render: PropTypes.func.isRequired
+        })).isRequired,
+        defaultTab: PropTypes.number
     };
 
     static defaultProps = {
-        children: null,
-        defaultTab: null
+        defaultTab: 0
     };
 
     __construct(props) {
         super.__construct(props);
 
-        const { defaultTab, tabNames } = this.props;
+        const { defaultTab } = this.props;
 
         this.state = {
-            activeTab: defaultTab || tabNames[0]
+            activeTab: defaultTab || 0
```

Comment:
Default prop for `defaultTab` is already a `0`, do we really need to check the second time there?